src/olympia/addons/migrations/0051_remove_accessibility.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 4.2.13 on 2024-05-22 11:39
+
+from django.db import migrations, models
+from olympia import amo
+from olympia.constants.categories import CATEGORIES
+
+
+def remove_accessibility_category(apps, schema_editor):
+    Addon = apps.get_model('addons', 'Addon')
+    AddonCategory = apps.get_model('addons', 'AddonCategory')
+    other_category_id = CATEGORIES[amo.ADDON_EXTENSION]['other'].id
+    old_accessibility_category_id = 74
```

Comment:
Speaking of old ids, could probably derive this ID from a query. What if there is ever another category with id 74 and then this migration is run again... not saying that is likely, but weirder things have happened.

Reply:
Category IDs are only defined in constants (and the constant is gone at this point) so we can't really query for that.

For constants of things we keep in db, like activity action ids, we keep a constant in the file and prefix it (and have a test making sure activity action ids are all unique), but here since I'm getting rid of all relevant `AddonCategory` rows entirely I figured I wouldn't need to worry about that. I'm open to alternatives though.